### PR TITLE
Moves nav items from top of page to left nav

### PR DESCRIPTION
**Pull request by intelkevinputnam**
1. Uses a redirect to home.html to replace index.html.
2. Moves home.rst to build location in conf.py
3. Replaces index.html with redirect version in conf.py

Signed-off-by: Kevin Putnam <kevin.putnam@intel.com>

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -87,8 +87,8 @@
 html_context = {
     'current_language': 'English',
     'languages': (('English', '/latest'), ('Chinese', '/cn/latest')),
-    'doxygen_mapping_file': '@DOXYGEN_MAPPING_FILE@',
-    'doxygen_snippet_root': '@OpenVINO_SOURCE_DIR@'
+    'doxygen_mapping_file': '',
+    'doxygen_snippet_root': '/home/kputnam/ov2'
```
Comment (totoka-intel) [inline docs/conf.py:91]: Isn't this hardcoded path here left by the mistake?

File: docs/conf.py [hunk 2]
```diff
@@ -164,12 +164,19 @@ def autodoc_skip_member(app, what, name, obj, skip, options):
     return name in exclude_pyapi_methods
 
 
+shutil.copy("../../../docs/home.rst",".")
+
+def replace_index_with_redirect(app,exception):
+    print(os.getcwd())
```
Comment (totoka-intel) [inline docs/conf.py:170]: What is the intention of this print? Is it helpful in other way than just debug this function?

File: docs/conf.py (hunk 1 above)
Comment (intelkevinputnam) [reply docs/conf.py:91]: Good catch. My shortcut caught up with me. I'll fix it.

File: docs/conf.py (hunk 2 above)
Comment (intelkevinputnam) [reply docs/conf.py]: Thanks for catching these. Should be fixed now.

**Pull request merged**